ci-operator/jobs/stolostron/application-ui-test/OWNERS

```diff
@@ -8,19 +8,4 @@ approvers:
 - vboulos
 options: {}
 reviewers:
-- birsanv
```

Comment:
If we remove these now they will just be adjusted automatically by the bot do we even want to bother removing these OWNERS?

Reply:
Do we need to add them back in the PR? Or the bot will handle it?
If the bot will do it then fine let it do it :)

Reply:
You can just update this PR to not remove them as a part of this PR

Reply:
cool, let the bot do it :)